_posts/en/newsletters/2019-08-07-newsletter.md

```diff
@@ -0,0 +1,129 @@
+---
+title: 'Bitcoin Optech Newsletter #58'
+permalink: /en/newsletters/2019/08/07/
+name: 2019-08-07-newsletter
+slug: 2019-08-07-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces the maintenance release of Bitcoin Core
+0.18.1, summarizes a discussion about BIP174 extensions, and notes a
+proposal for LN trampoline payments.  Also included are our regular
+sections on bech32 sending support and notable changes in popular
```

Comment:
Perhaps worth explicitly calling out the interview / field report from BRD in the summary as it is pretty compelling and notable.